src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -5470,4 +5470,7 @@ This option should be used when specifying the certificate via -CertificateSubje
   <data name="VerifyCommandAllDescription" xml:space="preserve">
     <value>Specifies that all verifications possible should be performed to the package(s).</value>
   </data>
+  <data name="VerifyCommandFingerprintAlgorithmDescription" xml:space="preserve">
+    <value>Hash algorithm to be displayed as certificate fingerprint in the certificate information. Defaults to SHA256.</value>
```

Comment:
Consider - 

`Hash algorithm used to generate the certificate fingerprints specified in the '-CertificateFingerprint' option. Defaults to SHA256.`

Reply:
This option is not intended to be just to be used with the `-CertificateFingerprint` option. One takes some assumptions from the other, but the user could provide one with the other without a problem.